onyo/lib/onyo.py

```diff
@@ -435,13 +435,15 @@ def fsck(self, tests: Optional[list[str]] = None) -> None:
         - "asset-yaml": loads each assets and checks if it's valid YAML
         - "asset-validity": loads each asset and validates the contents against
           the validation rulesets defined in ``.onyo/validation/``.
+        - "no-pseudo-keys": verifies that assets do not contain pseudo-key names
```

Comment:
IMO, the `no` should be dropped. It's a check for pseudo keys.

Have "no" in the name confuses the fact that "yes" we are doing the check.